packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -147,12 +154,13 @@ export interface IGarbageCollector {
     readonly summaryStateNeedsReset: boolean;
     /** Tells whether GC data should be written to the root of the summary tree. */
     readonly writeDataAtRoot: boolean;
+    readonly trackGCBlobState: boolean;
     /** Run garbage collection and update the reference / used state of the system. */
     collectGarbage(
         options: { logger?: ITelemetryLogger; runGC?: boolean; runSweep?: boolean; fullGC?: boolean; },
     ): Promise<IGCStats>;
     /** Summarizes the GC data and returns it as a summary tree. */
-    summarize(): ISummaryTreeWithStats | undefined;
+    summarize(fullTree: boolean, trackState: boolean): ISummaryTreeWithStats | undefined;
```

Comment:
Optional suggestion: create a type or interface `{ fullTree: boolean, trackState: boolean }` and take that type here.  Some benefits:

* It's self-documenting
* It's more robust to changes in the set of options (not brittle to parameter ordering, optional v. required, etc)

Reply:
I agree, this should be done as a type or interfaces. Eventually, I'm hoping that `trackState` gets removed. It'll be easier to find all the `summarize` methods with `trackState` if it looks consistent like this across all `summarize` calls.